src/components/Header/Header.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Try to solve the problem, not disable eslint
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md